MAUI/Cartesian-Charts/Trackball.md

```diff
@@ -220,6 +221,189 @@ this.Content = chart;
+                      XBindingPath="Name"
+                      YBindingPath="Values"/>
+
+    <chart:LineSeries ItemsSource="{Binding Data2}" 
+                      TrackballLabelTemplate ="{StaticResource trackballLabelTemplate}"
+                      XBindingPath="Name"
+                      YBindingPath="Values"/>
+
+    <chart:LineSeries ItemsSource="{Binding Data3}" 
+                      TrackballLabelTemplate ="{StaticResource trackballLabelTemplate}"
+                      XBindingPath="Name"
+                      YBindingPath="Values"/>
+                      
+</chart:SfCartesianChart>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfCartesianChart chart = new SfCartesianChart();
+
+ChartTrackballBehavior trackball = new ChartTrackballBehavior();
+chart.TrackballBehavior= trackball;
+. . .
+LineSeries series1 = new LineSeries();
+series1.ItemsSource = new ViewModel().Data1;
+series1.XBindingPath = "Name";
+series1.YBindingPath = "Values";
+
+LineSeries series2 = new LineSeries();
+series2.ItemsSource = new ViewModel().Data2;
+series2.XBindingPath = "Name";
+series2.YBindingPath = "Values";
+
+LineSeries series3 = new LineSeries();
+series3.ItemsSource = new ViewModel().Data3;
+series3.XBindingPath = "Name";
+series3.YBindingPath = "Values";
+
+DataTemplate trackballLabelTemplate = new DataTemplate(() =>
+{
+    HorizontalStackLayout horizontalStackLayout = new HorizontalStackLayout();
+
+    Label label = new Label
+    {
+        FontSize = 20,
+        VerticalOptions = LayoutOptions.Center,
+        TextColor = Colors.White
+    };
+    label.SetBinding(Label.TextProperty, "DataItem.Values");
+
+    Image image = new Image
+    {
+        Source = "image.png",
+        WidthRequest = 40,
+        HeightRequest = 40
+    };
+
+    horizontalStackLayout.Add(label);
+    horizontalStackLayout.Add(image);
+
+    return horizontalStackLayout;
+});
+
+series1.TrackballLabelTemplate = trackballLabelTemplate;
+series1.TrackballLabelTemplate = trackballLabelTemplate;
+series1.TrackballLabelTemplate = trackballLabelTemplate;
+
+chart.Series.Add(series1);
+chart.Series.Add(series2);
+chart.Series.Add(series3);
+this.Content = chart;
+        
+{% endhighlight %}
+
+{% endtabs %}
+
+![Trackball Label Template support in MAUI chart](Trackball_images/maui_chart_trackball_label_template.png)
+
+## Customize the Axis Label with DataTemplate
+
+Customize the appearance of axis label of trackball using [TrackballLabelTemplate[]() property of [ChartAxis](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.ChartAxis.html).
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<chart:SfCartesianChart>
+    <chart:SfCartesianChart.Resources>
+            <DataTemplate x:Key="axisLabelTemplate">
+                <HorizontalStackLayout>
+                    <Label WidthRequest="50" HeightRequest="20" HorizontalTextAlignment="Center" BackgroundColor="Blue" Text="{Binding ValueX}" TextColor="White" FontSize ="15"/>
```

Comment:
@EzhilarasanElangovan31 Only Label used in HorizontalStackLayout for template. Why need to use HorizontalStackLayout? You can directly use Label alone

Reply:
Removed HorizontalStack layout